tools/wasme/cli/docs/content/tutorial_code/wasme_operator.md

```diff
@@ -279,8 +287,64 @@ spec:
 EOF
 {{< /highlight >}}
 
-Try the request again:
+You can check the envoyfilter `productpage-v1-bookinfo-custom-filter.bookinfo` in bookinfo namespace:
+
+```
```

Comment:
Instead of using backticks for the codeblock here, I think we should use the highlight functionality, as the resulting yaml is non-trivial and hasn't be exposed to the tutorial-reader until this point. We should highlight the line with the updated value on it.

It should look something like this:

`{{< highlight yaml "hl_lines=40" >}}`
yaml goes here
`{{< /highlight >}}`

Reply:
done